pkg/yurtctl/cmd/revert/edgenode.go

```diff
@@ -214,24 +215,40 @@ func (r *RevertEdgeNodeOptions) RunRevertEdgeNode() (err error) {
 			if foundAutonomy {
 				delete(node.Annotations, constants.AnnotationAutonomy)
 			}
-
 			delete(node.Labels, projectinfo.GetEdgeWorkerLabelKey())
 			if _, err = r.clientSet.CoreV1().Nodes().Update(context.Background(), node, metav1.UpdateOptions{}); err != nil {
 				return err
 			}
 		} else {
-			klog.Errorf("Cannot do the revert, the worker node: %s is not a Yurt edge node.", node.Name)
-			return err
+			return fmt.Errorf("Cannot do the revert, the worker node: %s is not a Yurt edge node.", node.Name)
 		}
 		klog.Info("label openyurt.io/is-edge-worker is removed")
+		defer func() {
```

Comment:
how about move `remove edge label` routine to the end of `RunRevertEdgeNode ` func and after `RevertKubelet 
` and `RemoveYurthub `. because we should try to keep the reverse order of `RunConvertEdgeNode`.